courses/free-courses-en.md

```diff
@@ -970,8 +970,23 @@
 
 #### Svelte
 
+* [Learn Svelte](https://www.youtube.com/playlist?list=PLA9WiRZ-IS_ylnMYxIFCsZN6xVVSvLuHk) - Joy of Code
+* [Learn Svelte Full Course](https://www.youtube.com/watch?v=UGBJHYpHPvA) - Lihau Tan, freeCodeCamp
 * [Learn the Svelte JavaScript Framework](https://www.youtube.com/watch?v=ujbE0mzX-CU) - Noah Glaser, freeCodeCamp
+* [Svelte Crash Course](https://www.youtube.com/watch?v=3TVy6GdtNuQ) - Traversy Media
 * [Svelte Tutorial](https://www.youtube.com/watch?v=vhGiGqZ78Rs) - Beau Carnes, freeCodeCamp
+* [Svelte Tutorial for Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwiYZOsc2D8AO1MYwLjZQrKx) - Codevolution
+* [Svelte Tutorial for Beginners](https://www.youtube.com/playlist?list=PL4cUxeGkcC9hlbrVO_2QFVqVPhlZmz7tO) - The Net Ninja
+
+
+#### SvelteKit
+
+* [Beginner SvelteKit](https://www.youtube.com/playlist?list=PLtgYhHmUIr3qDB2eTzY-nuBH1W5tOK8a4) - Steph Dietz, Vercel
+* [Learn How To Build Modern Web Apps With SvelteKit](https://www.youtube.com/watch?v=MoGkX4RvZ38) - Joy of Code
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes